rpm/centos-7/Dockerfile

```diff
@@ -21,6 +21,17 @@ ENV SUITE=${SUITE}
 # failing, so replace the URL.
 RUN if [ -f /etc/yum.repos.d/CentOS-Sources.repo ]; then sed -i 's/altarch/centos/g' /etc/yum.repos.d/CentOS-Sources.repo; fi
 RUN yum install -y rpm-build rpmlint
+
+# Install Linux kernel headers for btrfs support.
+# See https://github.com/moby/moby/pull/44761 and https://github.com/moby/moby/pull/44762,
+# See https://www.kernel.org/doc/Documentation/kbuild/headers_install.txt
+RUN yum install -y make gcc \
+ && curl -fsSL -O https://github.com/torvalds/linux/archive/refs/tags/v4.12.tar.gz \
+ && tar -xvzf ./v4.12.tar.gz \
+ && rm v4.12.tar.gz \
+ && cd linux-4.12/ \
+ && make headers_install ARCH=arm64 INSTALL_HDR_PATH=/usr
```

Comment:
It seems like we shouldn't be setting ARCH here, should we? I don't quite see how this Dockerfile is ARM-specific.